<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSASystem</class>
 <widget class="QMainWindow" name="RSASystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOperate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Operate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClean">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>751</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>731</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QRadioButton" name="rbDecrypt">
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblAlphabet">
        <property name="text">
         <string>     Alphabet</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtPrime2"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtMessage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblMessage">
        <property name="text">
         <string>Message    </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPrime1">
        <property name="text">
         <string>Prime (N)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPrime1"/>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cmbAlphabet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="rbEncrypt">
        <property name="text">
         <string>Encrypt              </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPrime2">
        <property name="text">
         <string>Prime (N)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtPrime3"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblPrime3">
        <property name="text">
         <string>Prime (N)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RSA SYSTEM ENCRIPTION</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>751</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="QPushButton" name="btnSTM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Send to Message</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblResult">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Decrypted message</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="txtResult"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPrivKey">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblPrivKey">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Private Key (d)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
